### DISCUSSION TITLE
now build error with 'babel-plugin-module-resolve'

### DISCUSSION THREAD

**Discussion by emilforsmann** [vercel/next.js#12599] [Help] [answered]
Hi guys,

Is anybody else experiencing an issue with using babel-plugin-module-resolver on zeit/now?

getting this error: `Module not found: Can't resolve '../../../layout/main' in '/zeit/7b754faf/pages/[id]/settings'`

Running `build` locally or running `now dev` works like a charm. 
I've also tried removing package-lock.json and node_module and doing a clean build, the problem still remains. 

This is the specific module on pages/[id]/settings:
`import Main from 'layout/Main';`
and this is how module-resolver is specified:
```json
{
    "root": ["./"],
    "alias": {
     "components": "./components",
     "contexts": "./contexts",
     "hooks": "./hooks",
     "hoc": "./hoc",
     "lib": "./lib",
     "layout": "./layout",
     "mutations": "./mutations",
     "queries": "./queries"
    }
  }
```

**Comment by timneutkens** [comment, marked as answer]
You can use https://nextjs.org/docs/advanced-features/module-path-aliases instead which is already landed on stable.
Main benefit of the approach in the docs is that it also integrates with your editor.

**Reply by emilforsmann** [reply]
Thanks for the help Tim!
By switching to build in module-path-aliases and re-importing the component, the build error disappeared.
